uikit/views/src/main/java/com/sygic/maps/uikit/views/navigation/infobar/Infobar.kt

```diff
@@ -0,0 +1,286 @@
+ * altitude etc.) and as user interaction component. It contains two [ImageButton]'s (left/right) and pre-customized
+ * primary/secondary [TextView] which can be controlled with the [InfobarItemsHolder] class.
+ *
+ * The [Infobar] design can be completely changed with the custom _infoBarStyle_ or the standard android
+ * attributes as _background_, _navigationTextColorPrimary_ or _navigationTextColorSecondary_ can be used. The [Infobar]
+ * view has also own attributes such as _leftButtonIcon_, _leftButtonIconTint_, _leftButtonBackground_, _leftButtonBackgroundTint_,
+ * _rightButtonIcon_, _rightButtonIconTint_, _rightButtonBackground_ and _rightButtonBackgroundTint_.
+ */
+@Suppress("unused", "MemberVisibilityCanBePrivate")
+open class Infobar @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = R.attr.infoBarStyle,
+    defStyleRes: Int = R.style.SygicInfobarStyle
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private val binding: LayoutInfobarInternalBinding =
+        LayoutInfobarInternalBinding.inflate(LayoutInflater.from(context), this, true)
+
+    private var layoutMargin: Int = 0
+    private var layoutMarginTop: Int = 0
+    private var layoutMarginBottom: Int = 0
+    private var layoutMarginStart: Int = 0
+    private var layoutMarginEnd: Int = 0
+
+    init {
+        isClickable = true
+        layoutTransition = LayoutTransition()
+
+        attrs?.let { attributeSet ->
+            @Suppress("Recycle")
+            context.obtainStyledAttributes(
+                attributeSet,
+                R.styleable.Infobar,
+                defStyleAttr,
+                defStyleRes
+            ).also {
+                setBackgroundResource(it.getResourceId(R.styleable.Infobar_android_background, 0))
+
+                setLeftButtonImageResource(
+                    it.getResourceId(R.styleable.Infobar_leftButtonIcon, 0),
+                    it.getResourceId(R.styleable.Infobar_leftButtonIconTint, 0)
+                )
+                setLeftButtonBackgroundResource(
+                    it.getResourceId(R.styleable.Infobar_leftButtonBackground, 0),
+                    it.getResourceId(R.styleable.Infobar_leftButtonBackgroundTint, 0)
+                )
+                setRightButtonImageResource(
+                    it.getResourceId(R.styleable.Infobar_rightButtonIcon, 0),
+                    it.getResourceId(R.styleable.Infobar_rightButtonIconTint, 0)
+                )
+                setRightButtonBackgroundResource(
+                    it.getResourceId(R.styleable.Infobar_rightButtonBackground, 0),
+                    it.getResourceId(R.styleable.Infobar_rightButtonBackgroundTint, 0)
+                )
+
+                elevation = it.getDimensionPixelSize(R.styleable.Infobar_android_elevation, 0).toFloat()
+                layoutMargin = it.getDimensionPixelSize(R.styleable.Infobar_android_layout_margin, 0)
```

Comment:
menil som to v mojom PR - je treba tu mat -1 ako default, pretoze 0 je validna ked chces napr prebit marginy z def stylu. Aj v sourcoch ViewGroup sa pouziva -1 btw...

Reply:
https://github.com/Sygic/sygic-maps-kit-android/pull/76/files#diff-ad2c847f72e6ad617826c51cc8dd4820

Reply:
👍